ccd-definition/CaseField.json

```diff
@@ -428,6 +428,14 @@
     "FieldType": "JudgeAndLegalAdvisor",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "allocatedJudgeLabel",
```

Comment:
Instead of a new case field we should add a new field judgeLabel to AllocatedJudge complex type which is set in Allocated Judge event. Then we can use interpolation to access this field in CCD and show it within the collection. Current implementation would show it at the top of the page. Maybe should be separate a PR; but if we introduce this new case field and plan to remove it, will it cause some problems?

Reply:
I think this is important, we shouldn't make users scroll up to the top of the screen to view the judgeLabel if we have the ability to add the label in a better location

Reply:
This was the suggested approach for now.